src/components/organisms/UiList/_internal/UiListItemSuffix.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <UiButton class="ui-button--text ui-button--small ui-list-item-suffix">
+    <!-- @slot Use this slot to replace label template. -->
+    <slot
+      name="label"
+      v-bind="{ label }"
+    >
+      {{ label }}
+    </slot>
+    <!-- @slot Use this slot to replace icon template. -->
+    <slot
+      name="icon"
+      v-bind="{
+        hasIcon,
+        iconSuffixAttrs: defaultProps.iconAttrs
+      }"
+    >
+      <UiIcon
+        v-if="icon"
+        v-bind="defaultProps.iconAttrs"
+        class="ui-list-item-suffix__icon ui-button__icon "
+      />
+    </slot>
+  </UiButton>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import type { PropType } from 'vue';
+import type { PropsAttrs } from '../../../../types/attrs';
+import type { Icon } from '../../../../types/icon';
+import UiButton from '../../../atoms/UiButton/UiButton.vue';
+import UiIcon from '../../../atoms/UiIcon/UiIcon.vue';
+
+const props = defineProps({
+  /**
+   * Use this props to set label.
+   */
+  label: {
+    type: String,
+    default: '',
+  },
+  /**
+   * Use this props to set icon.
+   */
+  icon: {
+    type: [
+      String,
+      Object,
+    ] as PropType<Icon>,
+    default: 'info',
+  },
+  /**
+   * Use this props to pass attrs for UiIcon
+   */
+  iconAttrs: {
+    type: Object as PropsAttrs,
+    default: () => ({}),
+  },
+});
+const defaultProps = computed(() => ({
+  iconAttrs: {
+    icon: props.icon as Icon,
+    ...props.iconAttrs,
+  },
+}));
+const hasIcon = computed(() => !!defaultProps.value.iconAttrs.icon);
+</script>
+
+<style lang="scss">
+@use "../../../../styles/functions";
+
+.ui-list-item-suffix {
+  $element: list-item-suffix;
+
+  --button-icon-margin: #{functions.var($element + "-icon", margin, 0)};
+
+  display: flex;
+  gap: functions.var($element, gap, var(--space-4));
```

Comment:
```suggestion
```
IMO if we want to made some changes here we should make it in UiButton :). In this case you should use `ui-button__icon--right`

Reply:
Probably I miss this part in UiMenuItemSuffix 🙈